src/cdn/beer.ts

```diff
@@ -190,34 +174,50 @@ function updateFile (target: Element, e?: KeyboardEvent): void {
 function updateRange (target: Element): void {
   const parentTarget = parent(target) as HTMLElement;
   const bar = query("span", parentTarget) as HTMLElement;
-  const inputs = queryAll("input", parentTarget) as NodeListOf<HTMLInputElement>;
-  const tooltip = query(".tooltip", parentTarget) as HTMLElement;
+  const inputs = queryAll("input", parentTarget) as NodeListOf<any>;
```

Comment:
I was trying to avoid some casting in lines 196, 197 and 198. But I think the cast is not too bad in this case. (https://github.com/beercss/beercss/commit/2c3dd60c2555e0ce95bc4cc1dbd561df9dff98ed)